tox.ini

```diff
@@ -17,21 +17,18 @@ deps =
     -rtest-requirements.txt
     ansible22: ansible==2.2.1.0
 commands =
-    unit: py.test -vv -x --cov={toxinidir}/molecule/ {posargs}
+    unit: py.test -vv --cov-report=term-missing --cov={toxinidir}/molecule/ --no-cov-on-fail {posargs}
```

Comment:
- Remove -x so that other unit tests can run even if one fails.
  - I wasn't sure if there was a specific reason for using -x here
- use term-missing for the coverage report, to get a list of uncovered lines
- add --no-cov-on-fail to prevent an additional failure for coverage when there is a test failure.